### PR TITLE
docs: simplifying content

### PR DESCRIPTION
**Pull request by Karrenbelt**
## Proposed changes

still mentioning of fetch in these sections:
- Trust minimisation
- Language Agnostic Definition
- Agent & component registry

## Fixes

If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce to agents-aea?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [ ] Lint and unit tests pass locally with my changes and CI passes too
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that code coverage does not decrease.
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: docs/glossary.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 This glossary defines a number of terms commonly used across the documentation. For the definitions of framework components consult the API docs.
 
-* **AEA (Autonomous Economic Agent)**: An AEA is "an intelligent agent acting on an owner's behalf, with limited or no interference, and whose goal is to generate economic value to its owner". AEAs are a special type of agent. [<a href="../index">more</a>]
+* **AEA (Autonomous Economic Agent)**: An AEA is "an intelligent agent acting on an owner's behalf, with limited or no interference, and whose goal is to generate economic value to its owner". AEAs are a special type of agent. [<a href="../agent-oriented-development">more</a>]
```
Comment (DavidMinarsch) [inline docs/glossary.md:3]: I think the original reference was correct as we define AEA on the index page.

File: docs/index.md [hunk 2]
```diff
@@ -1,33 +1,65 @@
...
+* A registry for sharing implementations of agent component designs  
 
 
 ## Why build with the AEA Framework?
 
 The AEA framework provides the developer with a number of features, which combined cannot be found anywhere else:
 
-* The peer-to-peer <a href="acn">agent communication network (ACN)</a> allows your AEAs to interact with all other AEAs over the public internet.
+* The peer-to-peer <a href="acn">agent communication network (ACN)</a> allows your AEAs to interact with other AEAs over the public internet.
+* The <a href="oef-ledger">Open Economic Framework</a> provides a search and discovery system that allows AEAs to locate each other.
```
Comment (DavidMinarsch) [inline docs/index.md:62]: This has no place here. We don't have the concept of OEF and will never have it

File: docs/agent-oriented-development.md [hunk 3]
```diff
@@ -1,49 +1,56 @@
...
-## Decentralisation
 
-Multi-Agent Systems (**MAS**) are inherently decentralised. The vision is of an environment in which every agent is able to directly connect with everyone else and interact with them without having to rely on a third party acting as an intermediary or match-maker. This is in direct contrast to centralised systems in which a single entity is the central point of authority, through which all interactions happen. The conventional client-server model is an example of a centralized architecture where clients interact with one another regarding specific services (e.g. communication, commerce) only through a server.
+## Decentralisation
 
-This is not to say that facilitators and middlemen have no place in a multi-agent system; rather it is the '_commanding reliance on middlemen_' that MAS rejects.
+Allowing for direct peer-to-peer communication makes Multi-Agent Systems 
+(**MAS**) inherently decentralized. This contrasts conventional systems design, 
+such as that of the client-server model, where clients rely on message relay 
+services provided by an intermediary. That does not imply that facilitators of
```
Comment (DavidMinarsch) [inline docs/agent-oriented-development.md:15]:
```suggestion
such as that of the client-server model, where clients rely on a central point of authority, through which all interactions happen and on which full trust has to be placed. That does not imply that facilitators of 
```
```suggestion
such as that of the client-server model, where clients rely on message relay 
services provided by an intermediary. That does not imply that facilitators of 
```

File: docs/agent-oriented-development.md [hunk 4]
```diff
@@ -1,49 +1,56 @@
...
+**Division of responsibilities:** In a decentralised system every agent is equally privileged and (in principle) able to interact with any other agent. Without a central authority that provides the services needed to mediate agent interactions, hosting these becomes part of the agents' responsibility. An example hereof is the access to a registry of contact addresses that is normally maintained by a service provider. Without any form of government, however, a central database that can be updated at the sole discretion of any individual agent is easily corrupted. A system in which every agent is only responsible for maintaining their own set of records is also problematic, since errors and the asynchronous execution of events inevitably leads to a situation in which agents possess different versions of the record. Instead, agents must share responsibility over the state of records and do so by maintaining a local copy of it and periodically synchronizing this state by requiring consensus to be achieved over it.
```
Comment (DavidMinarsch) [inline docs/agent-oriented-development.md:19]: We lost the direct connection to DLT/blockchain on rhs which I think is unfortunate

File: docs/agent-oriented-development.md [hunk 5]
```diff
@@ -1,49 +1,56 @@
...
 
 ## Conflicting Environment
 
-As discussed above, the notion of decentralisation extends as far as ownership and control. Therefore, the different components that make up a decentralised system may each be owned by a different entity, designed according to very different principles and standards, with heterogeneous software and hardware, and each with internal objectives that may be fundamentally inconsistent, worse yet contradictory, with those of others. 
-
-As such, a distinctive characteristic of a multi-agent environment, is that it is inhabited by more than one agent (as the name suggests), where each agent may be owned potentially by a different stakeholder (individual, company, government). Since by design, each agent represents and looks after the interests of its owner(s), and because different stakeholders may have unaligned, conflicting, or contradictory interests, it is very common to have multi-agent systems in which the agents' objectives, values and preferences are unaligned, conflicting, or contradictory.
```
Comment (DavidMinarsch) [inline docs/agent-oriented-development.md:23]: I like some of the explicitness in this para. Could be moved up into the first part

**Comment by DavidMinarsch** [comment]
Please remove `sOEF` reference here: https://valory-xyz.github.io/open-aea/glossary/

Also remove: https://valory-xyz.github.io/open-aea/simple-oef/

This needs updating too: https://valory-xyz.github.io/open-aea/version/

**Comment by DavidMinarsch** [comment]
This also needs updating: https://valory-xyz.github.io/open-aea/vision/

**Comment by DavidMinarsch** [comment]
We should remove OEF throughtout: https://valory-xyz.github.io/open-aea/oef-ledger/

**Comment by DavidMinarsch** [comment]
Let's remove this: https://valory-xyz.github.io/open-aea/tac/

**Comment by DavidMinarsch** [comment]
I propose moving everything under `Development - Usecase` into a section under `Development - Intermediate`

File: docs/glossary.md (hunk 1 above)
Comment (Karrenbelt) [reply docs/glossary.md:3]: turns out you cannot reference index.md files like that, and need to use [<a href="../">more</a>] to reference there

**Comment by dvilelaf** [comment]
LGTM

File: docs/agent-oriented-development.md (hunk 4 above)
Comment (Karrenbelt) [reply docs/agent-oriented-development.md:19]: agreed, putting it back in

File: docs/agent-oriented-development.md (hunk 5 above)
Comment (Karrenbelt) [reply docs/agent-oriented-development.md:23]: rewritten and included

File: mkdocs.yml [hunk 6]
```diff
@@ -21,17 +21,17 @@ nav:
     - Agent-oriented development: 'agent-oriented-development.md'
     - Vision: 'vision.md'
     - Application areas: 'app-areas.md'
-    - Relation to OEF and Ledger: 'oef-ledger.md'
+    # - Relation to OEF and Ledger: 'oef-ledger.md'
```
Comment (DavidMinarsch) [inline mkdocs.yml:24]: Just delete the line and the file

File: mkdocs.yml [hunk 7]
```diff
@@ -21,17 +21,17 @@ nav:
...
   - Demos:
     - Demos: 'demos.md'
-    - Echo Demo: demos/echo_demo.md
-    - Gym example: 'gym-example.md'
-    - Gym skill: 'gym-skill.md'
-    - TAC external app: 'tac.md'
+    - Echo Demo: 'demos/echo_demo.md'
+    - Gym example: 'demos/gym-example.md'
+    - Gym skill: 'demos/gym-skill.md'
+    # - TAC external app: 'tac.md'
```
Comment (DavidMinarsch) [inline mkdocs.yml:34]: Just delete line and file

File: mkdocs.yml [hunk 8]
```diff
@@ -60,7 +60,7 @@ nav:
       - Configurations: 'config.md'
       - Limitations of v1: 'limits.md'
 
-  - Development - Use Case:
+  # - Development - Use Case:
```
Comment (DavidMinarsch) [inline mkdocs.yml:63]: Probably `Use Cases` could be the subsection under `Development - Intermeditate`

**Comment by DavidMinarsch** [comment]
> This also needs updating: https://valory-xyz.github.io/open-aea/vision/

I think you forgot to update the text here.

**Comment by DavidMinarsch** [comment]
> Please remove `sOEF` reference here: https://valory-xyz.github.io/open-aea/glossary/
> 
> Also remove: https://valory-xyz.github.io/open-aea/simple-oef/
> 
> This needs updating too: https://valory-xyz.github.io/open-aea/version/

The last one needs updating

File: docs/version.md [hunk 9]
```diff
@@ -1,8 +1,13 @@
...
+    The (open) AEA framework provides the necessary tools for creating 
+    Autonomous Economic Agents (AEA). It is a fork of the 
+    <a href="https://github.com/fetchai/agents-aea" target="_blank">legacy AEA 
+    framework</a> that gives no priority to any package vendor.
 
-<a href="https://img.shields.io/pypi/v/aea" target="_blank"><img alt="PyPI" src="https://img.shields.io/pypi/v/aea" /></a>. 
+The latest version of the Python implementation of the AEA Framework:
 
-The framework is under rapid development with frequent breaking changes in the run-up to version `1.0` which is due in Q1 2021.
+<a href="https://img.shields.io/pypi/v/aea" target="_blank"><img alt="PyPI" src="https://img.shields.io/pypi/v/aea" /></a>.
```
Comment (DavidMinarsch) [inline docs/version.md:10]: We need to change this to `open-aea`

File: mkdocs.yml [hunk 10]
```diff
@@ -60,10 +58,11 @@ nav:
       - Configurations: 'config.md'
       - Limitations of v1: 'limits.md'
 
-  - Development - Use Case:
-    - Front-end intergration: 'connect-a-frontend.md'
-    - HTTP Connection: 'http-connection-and-skill.md'
-    - Build an AEA on a Raspberry Pi: 'raspberry-set-up.md'
+  # - Development - Use Case:
```
Comment (DavidMinarsch) [inline mkdocs.yml:61]: Can you remove line 61 the comment?

**Review by DavidMinarsch — APPROVED** [review]

**Pull request merged**